rpcpb/rpc.proto

```diff
@@ -379,6 +386,14 @@ message AddPermissionlessValidatorResponse {
   ClusterInfo cluster_info = 1;
 }
 
+message AddPermissionlessDelegatorRequest {
+  repeated PermissionlessValidatorSpec validator_spec = 1;
```

Comment:
does it make sense to rename PermissioneslessValidatorSpec to PermissionlessStakerSpec?